.github/workflows/nightly-acceptance-tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Nightly Acceptance Tests
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  docker-build:
+    env:
+      OPERATOR_TARBALL_FILE: operator-image.docker.tar
+      CONFIGURATOR_TARBALL_FILE: configurator-image.docker.tar
+    outputs:
+      operator_tarball_file: ${{ env.OPERATOR_TARBALL_FILE }}
+      configurator_tarball_file: ${{ env.CONFIGURATOR_TARBALL_FILE }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.1.7
+
+      - uses: actions/setup-go@v5.0.2
+        with:
+          go-version-file: ./src/go/k8s/go.mod
+
+      - name: Install nix
+        uses: cachix/install-nix-action@v27
+
+      - name: Build
```

Comment:
If you copy [this](https://github.com/redpanda-data/helm-charts/blob/182bd51aeaaf2c53a86d9b5fd39a92be48ea5689/.github/workflows/test_redpanda.yaml#L7-L9) from the helm-charts repo, you can omit `nix develop -c bash` everywhere.